### PR TITLE
Add Pontoon links to QA views, improve accesskeys view

### LINKED ISSUE #870: Add the handy links to the pages in the QA View
When I use Access Keys and Translation Consistency views really miss the direct link to translation tool but suppose others can benefit from having them in the other views too.

### PR DESCRIPTION
**Pull request by flodolo**
Fixes #870 and #883. It's a first draft (seems to be working), but not ready for review yet.

The change to getEditLink is needed because I added a pontoon-mozorg to mozilla-l10n-query yesterday (for the webdashboard), since we have locales using Pontoon for mozilla.org but not for products (e.g. it, da, de, etc.).

Result: now we're showing Pontoon links to every locale.

### CODE REVIEW

**Comment by flodolo** [comment]
It should be ready for review.

**Comment by flodolo** [comment]
The accesskey logic was quite poor: 

1.  [Check](https://github.com/mozfr/transvision/blob/master/app/views/accesskeys.php#L24-L29) if the last 9 characters of a string ID are `accesskey`.
2. Remove 10 characters, add one of `['.label', '.title', '.title2']` and see if such a string ID exists.

I've already improve that logic on step 1, by searching for `accesskey` in the entire string (that would catch `accesskey2` for example), but I've realize we miss out on all DevTools access keys, where the schema is often:

```
foo = Some label
foo.accesskey = S
```

I need to think a bit more about this.

**Comment by flodolo** [comment]
Improved logic.

Esperanto moves from 17 to 24 access keys, there's even one for French.

File: app/models/accesskeys.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            if (in_array($entity, $ignored_ids)) {
+                continue;
+            }
+            /*
+                Store the string if the access key is empty or using a
+                character not available in the label.
+            */
+            if ($current_ak == '') {
+                $ak_results[$ak_string_id] = $entity;
+            } elseif (mb_stripos($target[$entity], $current_ak) === false) {
```
Comment (TheoChevalier) [inline app/models/accesskeys.php]: I assume it’s to avoid a (too) long condition in one `if`?

Comment (flodolo) [reply app/models/accesskeys.php]: No, this slipped through the various rewrites.

**Comment by flodolo** [comment]
> dir/lang attributes are missing on <div class="result_meta_link">, so for RTL locales, the <source> and <Pontoon> links are not displayed properly

Ugh, good catch. Fixed also the other view where I copied the code

**Pull request merged**